source/codegen/generate_server_impl_info.py

```diff
@@ -0,0 +1,43 @@
+"""Service information generation."""
+
+import argparse
+import json
+from pathlib import Path
+
+from template_helpers import load_metadata
+
+
+def _generate_file(metadata_dir: Path, output_dir: Path) -> None:
+    service_instance_names = []
+    driver_modules = [
+        load_metadata(p)
+        for p in sorted(metadata_dir.iterdir())
+        if p.is_dir() and "fake" not in p.name
+    ]
```

Comment:
Should we be limiting this like `stage_client_files` does to not include the restricted services? That stage_client_files checks the driver_readiness ('code_readiness' in config.py) through `common_helpers.get_driver_readiness` [here](https://github.com/ni/grpc-device/blob/a4430019dd911c664e3080804d6a6685b980d6db/source/codegen/stage_client_files.py#L24) to exclude any that aren't marked explicitly or implicitly with `'code_readiness': 'Release'`.

Reply:
Excellent point, I'll get on it.